src/components/clusterWizard/HostsTable.scss

```
@@ -0,0 +1,3 @@
+.host-table-alerts {
+  padding-top: 0.5rem !important;
```

I think we should use variable
```suggestion
  padding-top: var(--pf-global--spacer--lg) !important;
```

Also the file does not have to be `scss`, we most likely won't need any scss features and can use plain css.

---

Renamed